### PR TITLE
Add Lootcrate Icon

### LINKED ISSUE #7859: Request: Lootcrate
### Brand Name

Lootcrate

### Website

https://lootcrate.com/

### Popularity Metric

The Similarweb rank is 152,498 https://www.similarweb.com/website/lootcrate.com/#overview

### Official Resources for Icon and Color

None 

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by karlmathewramos**
simpleicons

**Issue:** closes #7859

**Similarweb rank:** 152498

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6438,6 +6438,11 @@
             "hex": "3F5DFF",
             "source": "https://loopback.io/resources"
         },
+        {
+            "title": "Lootcrate",
+            "hex": "111111",
```
Comment (LitoMore) [inline _data/simple-icons.json:6443]: I cannot find any `#111111` on their website. Could you provide a reference resource?

Comment (Karltheram) [reply _data/simple-icons.json:6443]: There is no hex source in the website so I used this website https://imagecolorpicker.com/en to get the hex value with the logo. Is this sufficient? If not I could change it to #000000 based on extracting the logo from wikipedia https://en.wikipedia.org/wiki/Loot_Crate

**Comment by LitoMore** [comment]
CleanShot 2022-10-03 at 18 33 28@2x

@Karltheram I see some differences. The expected shape should be like the blue one.

Please enable the aspect ratio lock when resize the shape. Thanks.

**Review by LitoMore — CHANGES_REQUESTED** [review]

**Comment by runxel** [comment]
Hi @karlmathewramos, are yout still on this?

File: _data/simple-icons.json [hunk 2]
```diff
@@ -6454,6 +6454,11 @@
             "hex": "3F5DFF",
             "source": "https://loopback.io/resources"
         },
+        {
+            "title": "Lootcrate",
+            "hex": "000000",
```
Comment (adamrusted) [inline _data/simple-icons.json:6459]:
```suggestion
            "hex": "1E1E1E",
```

Header uses the above hex code, we should stick with that.

File: _data/simple-icons.json [hunk 3]
```diff
@@ -6454,6 +6454,11 @@
             "hex": "3F5DFF",
             "source": "https://loopback.io/resources"
         },
+        {
+            "title": "Lootcrate",
```
Comment (adamrusted) [inline _data/simple-icons.json:6458]:
```suggestion
            "title": "Loot Crate",
```

The brand name is stylized as 'Loot Crate' on their website.

File: icons/lootcrate.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...2.388.339A678.009 678.009 0 0 1 1.996 21.27C-.034 20.8.018 19.206.018 18.067.018 14.122 0 10.6 0 6.865c0-1.186.057-1.222 1.104-1.88.966-.608 4.997-3.136 7.157-4.5.643-.406.961-.607 1.88-.406 1.88.41 7.75 1.65 10.91 2.339C23.476 2.947 24 3.31 24 4.744V8.38c0 .465-.008.638-.474.98-.235.174-1.568 1.14-2.345 1.688zm1.889-2.556c.083.019.164-.037.164-.112 0-1.414.013-2.495.013-3.666 0-.622-.408-1.137-1.304-1.343C17.235 2.29 10.24.866 9.897.778c-.23-.06-.375.108-.375.199 0 4.294.033 7.832.033 11.854 0 .74.475 1.525 1.503 1.765 4.165.973 7.65 1.655 12.014 2.667.084.019.166-.036.166-.112.002-.849.016-1.405.016-1.95 0-2.27.325-2.68-1.825-3.194-2.556-.612-6.171-1.378-6.458-1.44-.565-.125-.638-.267-.638-.592 0-.122-.013-2.346-.004-2.869.002-.157.007-.31.183-.397.156-.077.307-.046.772.053.52.111 5.347 1.186 7.786 1.73zm-8.897 14.735c.084.019.165-.036.165-.11.002-1.186.02-2.485.02-3.353 0-1.475.213-1.334-1.237-1.639-1.713-.36-5.155-1.146-6.939-1.552-.413-.094-.534-.424-.534-.998 0-1.726-.007-4.575-.007-6.577 0-1.834.224-1.599-.818-1.832-.657-.146-2.87-.637-3.852-.828-.167-.032-.261.085-.261.224-.004 3.94.061 8.21.061 11.994 0 1.228.51 1.815 1.536 2.051 4.173.963 7.496 1.652 11.866 2.62z"/></svg>
```
Comment (adamrusted) [inline icons/lootcrate.svg:1]: I'm seeing some differences - looks like the icon has been stretched to fit the square viewport. Could you please amend to keep the correct aspect ratio, as shown in red?

image

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Pull request closed without merging**